Ben's AD review comments (#52)
ietf-gitwg/using-github: draft-ietf-git-using-github.md

describes administrative processes that support the practices described in this
document.

Although the operation of IRTF Research Groups can be similar in function to

> Working Groups, this document only directly addresses the needs Working Groups.

Reviewer: ```suggestion
Working Groups, this document only directly addresses the needs of Working Groups.
```

A single organization SHOULD NOT be used for all IETF activity, or all activity
within an area. Large organizations create too much overhead for general
management tasks, particularly when there is a need to maintain membership.
management tasks.

Each organization requires owners.  The owner team for a Working Group
repository MUST include responsible Area Directors.  Area Directors MAY also
designate a delegate that becomes an owner and Working Group chairs MAY also be
owners.
GitHub requires that each organization have at least one owner.  The owners for
a Working Group repository MUST include responsible Area Directors.  Area
Directors MAY also designate a delegate that becomes an owner and Working Group
chairs MAY also be owners.

A team with administrator access SHOULD be created for the Working Group Chairs
and any Working Group Secretary.  Administrator access is preferable, since this
does not also include the ability to push to all repositories and ownership does

> not grant any other significant privileges.

Reviewer: I can understand that there's not a huge need to mention the details of Administrator privilege in these guidelines.

Author: Well, the point was that this was flat wrong, based on an outdated authorization model.

Reviewer 2: Should we mention that the model (as is now documented) is also subject to change?

within an area. Large organizations create too much overhead for general
management tasks.

GitHub requires that each organization have at least one owner.  The owners for
a Working Group repository MUST include responsible Area Directors.  Area

> Directors MAY also designate a delegate that becomes an owner and Working Group

Reviewer 2: ```suggestion
Directors MAY also designate a delegate that becomes an owner. Working Group
```

A simple example of how to do this is to include a link to the GitHub
organization on the WG Charter page in the datatracker.  Similarly, if there are

> additional resources, such as additional mailing lists, links to those resources

Reviewer 2: ```suggestion
additional resources, such as mailing lists, links to those resources
```

## Issue Tracking Mode

In addition to managing documents, the Working Group might choose to use GitHub
for tracking outstanding issues.  In this mode of interaction, a record of the
existence of substantive technical discussions is tracked using issues in the
issue tracker.  However, discussion of any substantial matters is always
conducted on mailing lists.

Under this mode, issues and pull requests can be opened by anyone, but anything
deemed substantive MUST be resolved exclusively on the mailing list.  Discussion on

> GitHub is kept to a recording the state of issues.  Only editorial matters can

Reviewer: ```suggestion
GitHub is limited to recording the state of issues.  Only editorial matters can
```
